sys/include/stdio_semihosting.h

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
do these defines need to be exported? If not, it is fine to define them in stdio_semihosting.c.

Reply:
Done!